back/app/services/json_schema_generator_service.rb

```diff
@@ -83,7 +83,8 @@ def visit_multiselect(field)
     {
       type: 'array',
       uniqueItems: true,
-      minItems: field.enabled? && field.required? ? 1 : 0,
+      minItems: field.enabled? && field.required? && !field.minimum_select_count? ? 1 : field.minimum_select_count || 0,
+      maxItems: field.maximum_select_count || field.options.count,
```

Comment:
I would recommend using `field.options.size` instead. This is potentially more efficient because`count` will always execute an SQL query, whereas `size` may not if the options have already been loaded (not sure if that's the case, but ideally it should be the case).